settings

```diff
@@ -21,6 +21,17 @@ load_settings() {
   # Expected: FULLGPGKEY
   # Optional: FULLVERSION
   . "$RELEASEDIR/settings"
+
+  if [[ $PROJECT != foreman ]]; then
+    if [[ -z "$FOREMAN_VERSION" ]]; then
+      echo "please set FOREMAN_VERSION in $RELEASEDIR/settings"
+      exit 1
+    elif [[ "$FOREMAN_VERSION" != "none" ]]; then
+      # load foreman settings, which will contain keys etc
+      . releases/foreman/settings
+      . "releases/foreman/${FOREMAN_VERSION}/settings"
```

Comment:
mhh, no, this overwrites OSes that have been set in `client/settings` before. damn.